homework/04LessonFour.js

```diff
@@ -77,6 +90,7 @@ var nestedObjects = {
   }
 };
 // update the starring property to have the value 'Tim Robbins'
+nestedObjects['users']['bob roberts']['starring'] = 'Tim Robbins';
```

Comment:
Great use of bracket notation!

![celebrate](https://media.giphy.com/media/l46CdUTy21h8wwrAs/giphy.gif)